package.json

```diff
@@ -12,6 +12,8 @@
         "build-docs": "cd src/docs/website && npm run build",
         "build-editor": "cd src/editor && npm run build",
         "build-server": "parcel build src/index.html",
+        "build-share": "parcel build src/share/export.html --public-url /share --out-dir dist/share",
+        "postbuild-share": "parcel build src/share/import.html --public-url /share --out-dir dist/share",
```

Comment:
Can you confirm if these are needed still?  Try just building the server `npm run build-server` and seeing if it also builds these share pages.